monitoring/grafana/dashboards/tests/features/environment.py

```diff
@@ -0,0 +1,119 @@
+# type: ignore[no-redef]
+# pylint: disable=E0611,W0613,E0102
+import copy
+
+from behave import given, then, when
+from termcolor import cprint
+
+from tests import PromqlTest
+from tests.util import get_dashboards_data, resolve_time_and_unit
+
+
+class GlobalContext:
+    def __init__(self):
+        self.tested_queries_count = 0
+        self.promql_expr_test = None
+        self.data = get_dashboards_data()
+        self.query_map = self.data['queries']
+
+    def reset_promql_test(self):
+        self.promql_expr_test = PromqlTest()
+        self.promql_expr_test.variables = copy.copy(self.data['variables'])
+
+    def print_query_stats(self):
+        total = len(self.query_map)
+        percentage = round((self.tested_queries_count / total) * 100, 2)
+        color = 'yellow'
+        if percentage > 90:
+            color = 'green'
+        cprint(f'Tested {self.tested_queries_count}/{total} {percentage}% of grafana queries',
+               color)
```

Comment:
This is a minor improvement but usually [test coverage stats are like this](https://coverage.readthedocs.io/en/6.1.1/):
```
File                      Expr   Miss  Cover   Missing
-------------------------------------------------------
host-overview.json            8      4    50%    17,23   # ine numbers from the JSON files (tricky) or panel-legend names
host-details.jons             8      8   100% 
-------------------------------------------------------
TOTAL                        16     12    75%
```

Reply:
done